packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -66,6 +66,37 @@ class Markdownz extends React.Component {
     return null
   }
 
+  findResizedImages (children) {
+    // const imageRegex = /(?:!\[(.*?)\]\((.*?)\))/g
+    const imageRegex = /!\[[^\]]+\]\([^)]+\)/g
+    const images = Array.from(children.matchAll(imageRegex))
+
+    let newChildren = children
+
+    if (images.length) {
+      images.forEach(img => {
+        const imgSizeRegex = /=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)/
```

Comment:
This regex will need an adjustment too because `markdown-it-imsize` allows only widths to be set like `=100x`. See how the images are being missed on this branch on TESS: 

<img width="1003" alt="Screen Shot 2021-05-14 at 3 10 03 PM" src="https://user-images.githubusercontent.com/5016731/118323646-89999480-b4c6-11eb-908d-50a7a7695063.png">

Here's the markdown string to work with:
`![exoplanet_cropped.jpg](https://panoptes-uploads.zooniverse.org/production/project_attached_image/38c81b00-5970-468b-9a91-8e0d102a8e82.jpeg =300x)`

I think adding `?` at the end should fix it: `/=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)?/` but we should add this to the example markdown and tests to be sure.

Reply:
Could this be placed inside `renderMedia()`, to transform `nodeProps.src`?

`\d+` might be too greedy. A real image size won’t have an infinite number of digits so you could limit the search to maybe 3 or 4 digits, max.

Reply:
Just caught up on the conversation elsewhere. So this can’t go in `renderMedia` because the URL is validated before that’s invoked?

Reply:
Correct. `remark` catches the image url with resize before as an invalid string before `renderMedia` can be invoked, as referenced here #1269.